src/npctalk.cpp

```diff
@@ -874,8 +881,11 @@ std::string dialogue::dynamic_line( const talk_topic &the_topic ) const
         alpha->shout();
         if( alpha->is_deaf() ) {
             return _( "&You yell, but can't hear yourself." );
-        } else {
-            return _( "&You yell." );
+            if( alpha->is_mute() ) {
```

Comment:
This code is dead, because prior statement is `return`.

Reply:
Thank you!  Did you fix this already as when I open npctalk.cpp in atom it looks like your changes?

Reply:
No, I did not fix anything. It is not my suggested code - just a place with error.

The suggestion to fix would be to add `        } else {` before `if( alpha->is_mute() ) {`